### PR TITLE
Fixed double fixed8 issue

### PR DESCRIPTION
**Pull request by ediopia**
There was double fixed8 balance on nep5.getTokenBalances()

### CODE REVIEW

**Comment by snowypowers** [comment]
hi Ed can we get some tests around this fix?

**Comment by ediopia** [comment]
Sorry for the late reply. When you see getTokenBalance() function right above the getTokenBalances(), you will see the same code as this fix which works fine.

**Comment by snowypowers** [comment]
having tests would be nice

**Comment by ediopia** [comment]
I added a test that compares a balance from from getTokenBalance().

File: packages/neon-nep5/__integration__/main.ts [hunk 1]
```diff
@@ -56,6 +56,13 @@ describe("getTokenBalances", () => {
     expect(Object.keys(balances)).toEqual(
       expect.arrayContaining(["RPX", "RHTT4", "LWTF", "NXT"])
     );
+    expect(balances.NXT instanceof u.Fixed8).toBe(true);
```
Comment (snowypowers) [inline packages/neon-nep5/__integration__/main.ts]: one test, one concern. Split it out into a separate test.

File: tsconfig-test.json [hunk 2]
```diff
@@ -10,3 +10,5 @@
     "packages/**/lib/**/*","packages/**/dist/**/*"
   ]
 }
+
```
Comment (snowypowers) [inline tsconfig-test.json]: dont need 2 newlines

**Comment by hal0x2328** [comment]
200 points ediopia
100 points snowypowers for review

**Comment by lllwvlvwlll** [comment]
https://neoscan.io/transaction/d82da2040b11e60522f6d409ad8f96d6842f527feb13941ce2fa787997ca06f7

**Pull request merged**